### DISCUSSION TITLE
vue3-easy-data-table__main has hardcoded `min-height`

### DISCUSSION THREAD

**Discussion by warflash** [HC200ok/vue3-easy-data-table#104] [Q&A] [answered]
Hey, 
I just noticed vue3-easy-data-table__main seems to have a hardcoded min-height of 180px.
https://github.com/HC200ok/vue3-easy-data-table/blob/29e8ed09919d2cf13fb11ffd5c1adff0106cd247/src/scss/vue3-easy-data-table.scss#L11

I know I can just overwrite it using deep css but that always feels sort of "dirty" so I was wondering if there's a reason for it to be there or if it's just a leftover of some sort^^

**Comment by HC200ok** [comment, marked as answer]
@warflash `min-height` works when loading data or there is no data. thank you for reminding me, I added a `table-min-height` prop (default 180) instead of a hardcode. please update to the latest version "1.4.16". FYI, there is another prop `table-height` you can use to make `vue3-easy-data-table__main` a certain height.

**Reply by warflash** [reply]
Awesome, thank you for the quick reply and patch, much appreciated! 🎉 It being used for the loading state makes a lot of sense, did not think of that tbh^^
